shadowapi/src/main/java/org/robolectric/util/ReflectionHelpers.java

```diff
@@ -202,7 +204,7 @@ public static <R> R getStaticField(Field field) {
    */
   public static <R> R getStaticField(Class<?> clazz, String fieldName) {
     try {
-      return getStaticField(getDeclaredField(clazz, fieldName));
+      return getStaticField(clazz.getDeclaredField(fieldName));
```

Comment:
What about `clazz` is null?

Reply:
To some extent this is just a revert of 4af252cce5f6d06f251e230d2c86becbdbddcb1c, which caused a scary warning when running tests. There has never been a null check for clazz here.